requirements.txt

```diff
@@ -1,3 +1,6 @@
 lupa==1.3
```

Comment:
@OHerasym, if you're switching to lunatic, then, please, remove lupa from requirements.txt and from readme